src/olympia/constants/base.py

```diff
@@ -233,8 +233,8 @@
 _size_tuple = namedtuple('SizeTuple', 'width height')
 # Preview upload sizes
 ADDON_PREVIEW_SIZES = {
-    'thumb': _size_tuple(200, 150),
-    'full': _size_tuple(700, 525)
+    'thumb': _size_tuple(320, 200),
+    'full': _size_tuple(1280, 800)
 }
```

Comment:
Should we add a comment where these sizes come from? An issue or a link to the frontend?

Reply:
https://github.com/mozilla/addons-server/pull/8553/commits/d002da8dc8c33a60929d9ab9fec9643d928c28c2